src/c03arrays/search/find_second_largest_element_in_array.js

```diff
@@ -0,0 +1,33 @@
+/* Write a program to find the second largest element in an array.
+Test Data:
+Input the size of array: 5
+Input 5 elements in the array:
+element - 0 : 2
+element - 1 : 9
+element - 2 : 1
+element - 3 : 4
+element - 4 : 6
+Expected Output:
+The Second largest element in the array is: 6 */
+
+
+function findSecondLargestElementInArray (array){
```

Comment:
I think this solution does not work correctly for arrays like [10,20] or [10,20,5,4,3,6]